gnomad/utils/annotations.py

```diff
@@ -1,10 +1,24 @@
 # noqa: D100
 
+import json
 import logging
 from typing import Any, Dict, List, Optional, Set, Tuple, Union
 
+import ga4gh
```

Comment:
```suggestion
import ga4gh.vrs as ga4gh_vrs
import ga4gh.core as ga4gh_core
```

Reply:
Kyle and I both ran into the problem where this loads, but for some reason ga4gh itself does not. Since it replicated I'm not overly worried about it, but would you have a better idea as how to format these two lines?